opengrok-tools/src/main/python/opengrok_tools/utils/log.py

```diff
@@ -25,6 +25,10 @@
 import sys
 
 
+def get_class_basename():
+    return __name__.split('.')[0]
```

Comment:
What if the name does not have a suffix?
Does it work with for example opengrok-indexer?

Reply:
Then it will take the basename which is fine. `opengrok-indexer` will create new `opengrok_tools` logger just like `opengrok-sync` et al.

Reply:
Actually all of the CLI tools have a suffix, e.g. if I insert `print(__name__)` in `indexer.py#main()`, it will report `opengrok_tools.indexer`.

Reply:
aha so this is to get the `opengrok_tools`